.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - 'node'
+  - '12'
```

Comment:
Changed travis to use a specific version of node

Looks like there was issues with `text-buffer` compiling on the latest version of node?

This change matches the version of node used by the current version of atom